routes/orderProducts.js

```diff
@@ -0,0 +1,48 @@
+const orderProductsRouter = require("express").Router();
+const { getOrderProductsById, addProductToOrder, updateOrderProduct, destroyOrderProduct} = require("../db")
+// const { isLoggedIn, isAdmin } = require("./util") need?
+
+
+orderProductsRouter.post("/orders/:orderId/products",  async (req, res, next) => {
+    const { orderId } = req.params;
+    const { productId, price, quantity } = req.body;
+    try {
+        const order_product = await addProductToOrder(orderId, productId, price, quantity)
+        res.send({
+            name: "Select item",
+            message: "Your item is added to cart."
+        }, order_product)
+    } catch(error) {
+        throw error;
+    };
+});
+
+orderProductsRouter.patch("/:orderProductId", async (req, res, next) => {
+    const { orderId } = req.params;
+    const { productId, price, quantity } = req.body;
+    try {
+        const updatedOrderProducts = await updateOrderProduct({ orderId, productId, price, quantity })
```

Comment:
Since you destructure the parameter as `id`, this has to be `updateOrderProduct({ id: orderProductId, productId, price, quantity })`

Reply:
also the `const` should be singular: `updateOrderProduct`

Reply:
updated and resolved.